randomize/plugin.py

```diff
@@ -1,6 +1,8 @@
 """
 This plugin randomizes the order of tests within a unittest.TestCase class
 
+This code is a fork of the nloadholtes/nose-randomize project (https://github.com/nloadholtes/nose-randomize)
```

Comment:
Quick note: If I merge this in it will look a little funny (since this code will be a part of the https://github.com/nloadholtes/nose-randomize repo)